packages/studio/src/components/PropsPanel.tsx

```diff
@@ -33,11 +33,13 @@ export default function PropsPanel(): JSX.Element | null {
           <Divider />
         </>
       )}
-      <ActiveComponentPropEditors
-        activeComponentState={extractedComponentState}
-        propShape={propShape}
-      />
-      <RepeaterPanel />
+      <div className="grow flex flex-col h-fill">
```

Comment:
what are these classes adding? I see they're in a couple components

Reply:
not sure about the others, but I found that the `grow flex` makes sure that the dropdowns aren't cut off
![image](https://github.com/yext/studio/assets/59857107/17055eee-6f9c-40fc-9497-2f5db6fbf14c)

Reply:
Yes, it makes the component extend as necessary so that the dropdown isn't cut off. It needs to be in the dropdown's parent components as well so that they also grow, otherwise the child can only grow to the height of the parent.